kbase-extension/static/kbase/js/widgets/appWidgets2/paramResolver.js

```diff
@@ -9,126 +114,53 @@ define(['bluebird', 'require'], (Promise, require) => {
             switch (dataType) {
                 case 'string':
                     switch (controlType) {
-                        case 'dropdown':
-                            return {
-                                input: 'selectInput',
-                                view: 'selectView',
-                            };
                         case 'textarea':
-                            return {
-                                input: 'textareaInput',
-                                view: 'textareaView',
-                            };
                         case 'file':
-                            return {
-                                input: 'fileInput',
-                                view: 'fileView',
-                            };
+                            return controlType;
```

Comment:
remove all the redundant data and create the full module names via interpolation in the `load(Input|View)Control` functions.